org.mixedrealitytoolkit.uxcore/Button/PressableButton.cs

```diff
@@ -474,7 +525,14 @@ protected override void OnHoverExited(HoverExitEventArgs args)
             {
                 // Remove from our valid poke hash set if it was registered there.
                 validPokeInteractors.Remove(pokeInteractor);
+
+                if (validPokeInteractors.Count == 0)
+                {
+                    isHovered = false;
+                }
             }
+
+            ProximityHoverExited?.Invoke(this, null, args);
```

Comment:
See my comment with the `UpdateProximityHovered()` suggestion....you should remove the event invoking here and call `UpdateProximityHovered()`

Reply:
Implemented in latest commit.  Resolving this.